src/index.html

```diff
@@ -11,7 +11,185 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post">
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="surname">Surname:</label>
+        <input
+      class="duo"
+      type="text"
+      id="surname"
+      name="info"
+      autocomplete="off"
+        >
+        <br>
+        <label for="name">Name:</label>
+        <input
+      class="duo"
+      type="text"
+      id="name"
+      name="information"
```

Comment:
The name attribute should be more descriptive, e.g. 'name' instead of 'information'.